pages/account/manage/links.js

```diff
@@ -105,7 +105,7 @@ export default function ManageLinks({ BASE_URL, username, links }) {
           additionalMessage="Your profile information has been saved successfully."
         />
 
-        <div className="flex gap-4">
+        <div className="flex gap-4 mb-5">
```

Comment:
I think this change is not related to this issue?

Reply:
This was to fix the consistency of the UI. But now removed.